NewForce-Cohort-4/nutshell-the-lucky-biscuits
# Jp event list

# Description
Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
Fixes # (issue)
## Type of change
Please delete options that are not relevant.

# Testing Instructions
Login to your account and click the "Add a new event" button. A modal will appear with name, date, and location input fields. Insert the event info and click "Save Event." The event that is occuring next will appear at the top of the list of events. Events with dates that have expired will not appear in the list but will still be stored in the database.
# Checklist:
